web/models/announcement.ex

```diff
@@ -44,6 +44,12 @@ defmodule Constable.Announcement do
     ]
   end
 
+  def without_interests(query, interests \\ []) do
+    from a in query,
+      full_join: i in assoc(a, :interests),
+      where: not i.name in ^interests or is_nil(i.name)
```

Comment:
That's a really good question, and I'm not sure I have a solid answer. I was thinking we could try including them for now and exclude them later if it ends up being a problem. Also thank you! :)